src/encoded/static/components/viz/Workflow/Node.js

```diff
@@ -11,8 +12,7 @@ import { Fade } from 'react-bootstrap';
 export class DefaultNodeElement extends React.Component {
```

Comment:
This Component/Class isn't used by us but only exists for if we publish to NPM. Will likely need a tiny bit of clean-up; along with saving SCSS into NPM lib. For later / low priority.